Source/WebGPU/WebGPU/Buffer.h

```diff
@@ -123,7 +123,9 @@ class Buffer : public WGPUBufferImpl, public RefCounted<Buffer>, public CanMakeW
     void incrementBufferMapCount();
     void decrementBufferMapCount();
 
+WebGPUSwift_private:
```

Comment:
How about we used private_begin(public:)/private_end(private:) instead of WebGPUSwift_private:/private:.
When Swift team comes around providing us with a better mechanism then we define all these macros to "" and we are done.. else we will be left with redundant `private:` specifiers which we manually have to go remove, although they are harmless:
```
private:

...
...

private:

...
...
private:

.....
```

Reply:
If we go with `protected: PUBLIC_IN_WEBGPU_SWIFT`, then we can just grep and delete all instances of PUBLIC_IN_WEBGPU_SWIFT once we have compiler support.